widgetastic_manageiq.py

```diff
@@ -878,3 +875,57 @@ def last_page(self):
     @property
     def items_amount(self):
         return self.paginator.page_info()[1]
+
+
+class AlertEmail(Widget):
+
+    ROOT = "//div[@id='edit_email_div']"
+    ADD_BUTTON = ".//div[@title='Add']"
+    RECIPIENTS = "./div[@id='edit_to_email_div']//a"
+    EMAIL_LINK = ".//a[text()='{}']"
+
+    def __init__(self, parent, logger=None):
+        Widget.__init__(self, parent, logger=logger)
+        self.RECIPIENTS_INPUT = TextInput(self, "email")
+
+    def __locator__(self):
+        return self.ROOT
+
+    def fill(self, values):
+        if isinstance(values, basestring):
+            values = [values]
+        elif isinstance(values, list):
+            pass
+        else:
+            raise ValueError("Emails should be a string or a list")
+        if self.read() == set(values):
```

Comment:
Please do not use read() inside fill() so the logs are not confusing. read() should usually read some property which has name appropriate to the context and just unifies the interface.